examples/nextjs-with-typescript/package.json

```diff
@@ -1,28 +1,28 @@
 {
-  "name": "nextjs-with-typescript",
-  "version": "5.0.0",
+  "name": "next-mui-emotion",
+  "version": "0.1.0",
   "private": true,
-  "dependencies": {
-    "@emotion/cache": "latest",
-    "@emotion/react": "latest",
-    "@emotion/styled": "latest",
-    "@emotion/server": "latest",
-    "@mui/material": "latest",
-    "clsx": "latest",
-    "next": "latest",
-    "react": "latest",
-    "react-dom": "latest"
-  },
   "scripts": {
-    "dev": "next",
+    "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "post-update": "echo \"codesandbox preview only, need an update\" && yarn upgrade --latest"
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@emotion/cache": "^11.6.0",
```

Comment:
We use "latest" here, so that we don't have to update the examples each time a version is bumped.